src/mpi/coll/cvars.txt

```diff
@@ -156,6 +156,61 @@ cvars:
         smp                                     - Force smp algorithm
         scatter_recursive_doubling_allgather    - Force Scatter Recursive-Doubling Allgather
         scatter_ring_allgather                  - Force Scatter Ring
+        pipelined_tree                          - Force tree-based pipelined algorithm
+        tree                                    - Force tree-based algorithm
+	
+    - name        : MPIR_CVAR_BCAST_TREE_KVAL
+      category    : COLLECTIVE
+      type        : int
+      default     : 2
+      class       : none
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        k value for tree (kary, knomial, etc.) based bcast
+
+    - name        : MPIR_CVAR_BCAST_TREE_TYPE
+      category    : COLLECTIVE
+      type        : string
```

Comment:
I don't remember, but why these are not an enum?

Reply:
This CVAR lets users select a tree type for Bcast. The values are defined to be consistent with Ibcast.